src/components/ImageSequence/ImageSequence.js

```diff
@@ -0,0 +1,244 @@
+import React, { memo, useLayoutEffect, useRef, useCallback, useEffect, useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import checkProps from '@jam3/react-check-extra-props';
+
+import './ImageSequence.css';
```

Comment:
```suggestion
import './ImageSequence.scss';
```
should be supported now